dokka-subprojects/plugin-base/src/main/resources/dokka/scripts/platform-content-handler.js

```diff
@@ -1,35 +1,6 @@
 /*
  * Copyright 2014-2024 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
  */
-/** When Dokka is viewed via iframe, local storage could be inaccessible (see https://github.com/Kotlin/dokka/issues/3323)
- * This is a wrapper around local storage to prevent errors in such cases
- * */
-const safeLocalStorage = (() => {
```

Comment:
SafeLocalStorage was moved to a separate script.
This was made to reduce duplication of it's declaration and to ensure that it's loaded before other scripts which depend on safeLocalStorage